packages/xod-cloud-tabtest/src/compile.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable prettier/prettier */
+import * as R from 'ramda';
+import btoa from 'btoa';
+import fetch from 'node-fetch';
+import { createError } from 'xod-func-tools';
+
+import arduinoH from 'xod-tabtest/cpp/Arduino.h';
+import arduinoCpp from 'xod-tabtest/cpp/Arduino.cpp';
+import xStringFormatInl from '../../../cpplib/catch2utils/XStringFormat.inl';
+
+import * as EC from './errorCodes';
+
+// :: String -> StrMap Source -> Promise
+const compileSuite = R.curry((hostname, suite) => {
+  const reqUrl = `https://api.${hostname}/compile/enqueue`;
+  const reqBody = R.compose(
+    JSON.stringify.bind(JSON),
+    filesMap => ({
+      fqbn: 'wasm:tabtest',
+      options: {
```

Comment:
Maybe put this to `vendor` dir with `runtime.js`? Because this is very static options.